### PR TITLE
Added alpha spacy support for Chinese tokenizer

### PR DESCRIPTION
**Pull request by SenWu**

### CODE REVIEW

File: docs/user/faqs.rst [hunk 1]
```diff
@@ -84,11 +84,17 @@ For OS X::
     $ brew install mecab-ipadic
 
 Afterwards, you can use ``pip install fonduer[spacy_ja]`` to install Fonduer
-with Japanese language support. If you would like to use other languages with
-spaCy alpha support, which are not yet integrated in Fonduer, feel free to open
-an Issue_.
+with Japanese language support.
+
+If you would like to use Fonduer for Chinese documents, you can use
+``pip install fonduer[spacy_ja]`` to install Fonduer with Chinese language support.
```
Comment (lukehsiao) [inline docs/user/faqs.rst]: This description is looks wrong? Should be `spacy_zh`? Both Chinese and Japanese should be described.

File: docs/user/faqs.rst [hunk 2]
```diff
@@ -84,11 +84,17 @@ For OS X::
...
+
+If you would like to use other languages with spaCy alpha support, which are
+not yet integrated in Fonduer, feel free to submit a `Pull requests`_ or open an Issue_.
```
Comment (lukehsiao) [inline docs/user/faqs.rst]:
```suggestion
not yet integrated in Fonduer, feel free to submit a `Pull Request`_ or open an Issue_.
```

File: docs/user/faqs.rst [hunk 3]
```diff
@@ -84,11 +84,17 @@ For OS X::
...
 
 .. _see spacy language support: https://spacy.io/usage/models#languages
 .. _see spacy alpha languages: https://spacy.io/usage/models#alpha-support
 .. _see mecab on PyPI: https://pypi.org/project/mecab-python3/
+.. _see jieba on PyPI: https://pypi.org/project/jieba/
+.. _Pull requests: https://github.com/hazyresearch/fonduer/pulls
```
Comment (lukehsiao) [inline docs/user/faqs.rst]:
```suggestion
.. _Pull Request: https://github.com/hazyresearch/fonduer/pulls
```

File: setup.py [hunk 4]
```diff
@@ -30,7 +30,10 @@
         "treedlib>=0.1.1, <0.2.0",
         "wand>=0.4.4, <0.5.0",
     ],
-    extras_require={"spacy_ja": ["mecab-python3>=0.7,<0.8"]},
+    extras_require={
+        "spacy_ja": ["mecab-python3>=0.7,<0.8"],
```
Comment (lukehsiao) [inline setup.py]:
```suggestion
        "spacy_ja": ["mecab-python3>=0.7, <0.8"],
```

File: docs/user/faqs.rst (hunk 1 above)
Comment (SenWu) [reply docs/user/faqs.rst]: Fixed! Thanks!

**Pull request merged**